source/localizable/routing/redirection.md

```diff
@@ -1,24 +1,34 @@
-Calling [`transitionTo()`][1] from a route or [`transitionToRoute()`][2] from a 
-controller will stop any transition currently in progress and start a new 
-one, functioning as a redirect. `transitionTo()` behaves exactly like the 
-[link-to](../../templates/links) helper.
+Sometimes you want to redirect a user to a different page than what they requested for.
 
-[1]: http://emberjs.com/api/classes/Ember.Route.html#method_transitionTo
-[2]: http://emberjs.com/api/classes/Ember.Controller.html#method_transitionToRoute
+For example, if they're not logged in, you might want to prevent them from editing their profile, accessing private information,
+or checking out items in their shopping cart.
+Usually you want to redirect them to the login page, and after they have successfully logged in, take them back to the page they originally wanted to access.
+Another use case could be preventing a user who has already logged in from visiting the login page again.
+
+There's many other reasons you probably want to have the last word on whether a user can or cannot access a certain page.
+Whatever the reason, Ember allows you that control with a combination of hooks and methods in your route.
+
+One of the methods is [`transitionTo`](http://emberjs.com/api/classes/Ember.Route.html#method_transitionTo).
+Calling `transitionTo` from a route or
+[`transitionToRoute`](http://emberjs.com/api/classes/Ember.Controller.html#method_transitionToRoute) from a controller will stop any transitions currently in progress and start a new one, functioning as a redirect.
+`transitionTo()` behaves exactly like the [link-to](../../templates/links) helper.
+
+The other one is [`replaceWith()`](http://emberjs.com/api/classes/Ember.Route.html#method_replaceWith) which works the same way as `transitionTo()`.
+The only difference between them is how they manage history.
+`replaceWith()` substitutes the current route entry and replaces it with that of the route we are redirecting to,
+while `transitionTo()` leaves the entry for the current route and creates a new one for the redirection.
 
 If the new route has dynamic segments, you need to pass either a _model_ or an _identifier_ for each segment.
-Passing a model will skip that segment's `model()` hook (since the model is
-already loaded).
+Passing a model will skip the route's `model()` hook (since the model is already loaded).
 
 ## Transitioning Before the Model is Known
 
-If you want to redirect from one route to another, you can do the transition in
-the [`beforeModel()`][3] hook of your route handler.
-
-[3]: http://emberjs.com/api/classes/Ember.Route.html#method_beforeModel
+Since a route's [`beforeModel()`](http://emberjs.com/api/classes/Ember.Route.html#method_beforeModel) executes before the `model()` hook,
+it's a good place to do a redirect if you don't need any information that is contained in the model.
 
 ```app/router.js
 Router.map(function() {
+  this.route('index', { path: '/' });
```

Comment:
Not sure that's useful here, I believe index routes are explained before. And it might lead people into thinking they have to.